### PR TITLE
[4] Remove release news from post installation messages screen

### PR DESCRIPTION
**Pull request by PhilETaylor**
Closes https://github.com/joomla/joomla-cms/issues/33312

### Steps to reproduce the issue

**This PR removes the RSS Feed loading on the Post Installation Messages page.**

### Reasoning behind this PR

When you view the "Post-installation Messages for Joomla CMS" page, **it is always slow-er to load** as it has to go out and retrieve and render the RSS feed of release news. 

This also doesn't appear to be cached. 

"Release news from the Joomla! Project" really has nothing to do with "Post-installation Messages for Joomla CMS"

When viewing "Post-installation Messages for Joomla CMS" there is no reason to be shown what the latest release news is, it has nothing to do with what the messages are.

It would make more sense that a CACHED version of the "Release news from the Joomla! Project" was a toggle-able module on the Home Dashboard (if it must be anywhere). 

Screenshot 2021-04-25 at 17 41 18

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on e1f849cf3b140aeeb407ed70c3d5879e55dc8863<br /><br />The PR does what it says.

Question to the community: Shall we add another place in backend where you can see the release notes feed?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33324">issues.joomla.org/tracker/joomla-cms/33324</a>.</sub>

**Comment by PhilETaylor** [comment]
> Question to the community: Shall we add another place in backend where you can see the release notes feed?

Joomla already alerts if a new version is available... that's good enough :)

File: administrator/components/com_postinstall/tmpl/messages/default.php [hunk 1]
```diff
@@ -10,30 +10,11 @@
 defined('_JEXEC') or die;
 
 use Joomla\CMS\Factory;
```
Comment (Quy) [inline administrator/components/com_postinstall/tmpl/messages/default.php:12]:
```suggestion
```

File: administrator/components/com_postinstall/tmpl/messages/default.php [hunk 2]
```diff
@@ -84,11 +60,3 @@
 	</div>
 	<?php endforeach; ?>
 <?php endif; ?>
-<?php if ($this->eid == $this->joomlaFilesExtensionId) : ?>
-	</div>
-	<div class="col-md-4">
-		<h2><?php echo Text::_('COM_POSTINSTALL_LBL_RELEASENEWS'); ?></h2>
```
Comment (Quy) [inline administrator/components/com_postinstall/tmpl/messages/default.php:90]: Please delete language string.

Comment (PhilETaylor) [reply administrator/components/com_postinstall/tmpl/messages/default.php:90]: Done thanks

**Comment by richard67** [comment]
after the last change I get a "class 'Factory' not found" error.

**Comment by PhilETaylor** [comment]
lol - I blindly trusted @quy :)

**Comment by richard67** [comment]
I don't even trust myself blindly 😉 .

**Comment by PhilETaylor** [comment]
Factory is needed - adding back

**Comment by Quy** [comment]
Oops. Sorry I see its usage now.

**Comment by PhilETaylor** [comment]
Refactored the deprecated method use too :) 

`<?php if (Factory::getApplication()->getIdentity()->authorise('core.edit.state', 'com_postinstall')) : ?>`

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 2dfb59d1f65def4e0e19f9095d6116d4e6b65b33<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33324">issues.joomla.org/tracker/joomla-cms/33324</a>.</sub>

**Comment by dgrammatiko** [comment]
@PhilETaylor you're deleting my awesome code from 2014 https://github.com/joomla/joomla-cms/pull/5222. Well done! kill it with fire 😂

**Comment by dgrammatiko** [comment]
I have tested this item :white_check_mark: successfully on 2dfb59d1f65def4e0e19f9095d6116d4e6b65b33<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33324">issues.joomla.org/tracker/joomla-cms/33324</a>.</sub>

**Comment by richard67** [comment]
@dgrammatiko He did not restore the iframe at least ;-)

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33324">issues.joomla.org/tracker/joomla-cms/33324</a>.</sub>

**Comment by Quy** [comment]
Thank you!

**Pull request merged**